StartEmailValidationProcess/handler.ts

```diff
@@ -99,13 +106,29 @@ export function StartEmailValidationProcessHandler(
     );
 
     const dfClient = df.getClient(context);
-    await dfClient.startNew(
-      "EmailValidationProcessOrchestrator",
-      undefined,
-      emailValidationProcessOrchestartorInput
-    );
-
-    return ResponseSuccessAccepted();
+    return taskEither
+      .of(makeStartEmailValidationProcessOrchestratorId(fiscalCode, email))
+      .chain(orchId =>
+        isOrchestratorRunning(dfClient, orchId).chain(_ =>
+          _.isRunning
+            ? taskEither.of(void 0)
```

Comment:
that would be better to chain fromPredicate here instead of using taskEither.of(void 0)